packages/core/admin/admin/src/content-manager/components/BlocksEditor/hooks/tests/useBlocksStore.test.js

```diff
@@ -268,23 +268,24 @@ describe('useBlocksStore', () => {
     expect(quote).toBeInTheDocument();
   });
 
-  // TODO fix this test
-  it.skip('renders a list block properly', () => {
+  it('renders an unordered list block properly', () => {
     const { result } = renderHook(useBlocksStore, { wrapper: Wrapper });
 
     render(
-      result.current.list.renderElement({
-        children: 'list item',
+      result.current['list-unordered'].renderElement({
+        children: 'list unordered',
         element: {
           format: 'unordered',
         },
         attributes: {},
-      })
+      }),
+      {
+        wrapper: Wrapper,
+      }
     );
 
     const list = screen.getByRole('list', 'list item');
     expect(list).toBeInTheDocument();
-    expect(list.tagName).toBe('ul');
```

Comment:
I removed this since other tests aren't testing tagName. I

If we want to add it back tagName's are capitalized: UL. I'm curious why though? Is that slate thing?

Reply:
Yes checking the tagname shouldn't be necessary since we're doing getByRole. No idea why it would be capitalized though